neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/common/impl/RemoteNeo4jConfiguration.java

```diff
@@ -150,4 +158,12 @@ public Long getEstablishConnectionTimeout() {
 	public boolean isAuthenticationRequired() {
 		return authenticationRequired;
 	}
+
+	/**
+	 * @see Neo4jProperties#CONNECTION_POOL_SIZE
+	 * @return the size of REST connection pool
+	 */
+	public Integer getClientRestPoolSize() {
```

Comment:
Remove the Rest part, you can call it ClientPoolSize().

Reply:
OK. thanks!